flamegraph.pl

```diff
@@ -682,16 +685,21 @@ sub flow {
 		}
 		
 		t.textContent = txt;
+
 		// Fit in full text width
-		if (/^ *\$/.test(txt) || t.getSubStringLength(0, txt.length) < w)
+		if (/^ *\$/.test(txt) || t.getComputedTextLength() < w)
```

Comment:
I'm actually using node-stackvis, but here are the changes I'm testing:
https://github.com/cwarden/node-stackvis/commit/9c8f97d92571ea4a2b1a45cabbfc3d52c058f1f6